src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -949,7 +949,8 @@ def test_multi_kernel_single_omp_region():
         assert (omp_end_do_idx - end_do_idx) == 1
 
 
-def test_multi_different_kernel_omp(monkeypatch, annexed):
+def test_multi_different_kernel_omp(
+        tmpdir, f90, f90flags, monkeypatch, annexed):
```

Comment:
Sorry to be a pain but, since you've modified this test already, please could you modify it some more to use the `dist_mem` fixture?

Reply:
Done. Very elegant solution using fixture, I must admit.

Reply:
Also updated the compilation testing after merging changes from the latest master.

Reply:
Great, thanks.